packages/docs/docs/guides/architecture.mdx

```diff
@@ -20,51 +20,76 @@ export const Highlight = ({ children, color, textColor }) => (
 
 :::note See Also
 
-- [Performance](./performance.md)
-- [AI+UI](./ai-ui.md)
+- [Performance](./performance.md) - Discusses trade-offs between performance and correctness. Presents five strategies for improving performance in your AI-powered apps.
+- [AI+UI](./ai-ui.md) - Covers how AI.JSX enables dynamically created UI ("Just in Time" UI) and the approaches that can be used.
+  :::
 
+AI.JSX runs in both NodeJS and the browser. It can fit in no matter how you are deploying your application. This guide covers the five architectural approaches we expect most AI-apps to use and also discusses the key trade-offs and considerations for each one.
+
+## Summary: Architectural Deployment Options
+
+The five architectural approaches (listed in order from most client-side logic to least) are:
+
+1. [Client-Side Only](#architecture-1-client-side-only) <Highlight color="rgb(230, 246, 230)" textColor="rgb(0, 49, 0)">Great for Hackathons</Highlight>
+1. [API Proxy](#architecture-2-api-proxy)
+1. [AI.JSX Server-Side + Client UI](#architecture-3-aijsx-server-side--client-ui) <Highlight color="#4cb3d4">Recommended</Highlight>
```

Comment:
```suggestion
1. [Server-Side AI.JSX + Client UI](#architecture-3-aijsx-server-side--client-ui) <Highlight color="#4cb3d4">Recommended</Highlight>
```

Minor: I think this might be a bit cleaner because it keeps the pattern consistent of `<AI.JSX | UI> <server | client>`.

Reply:
yes agreed this is cleaner. implementing this.